xinference/model/llm/llm_family.json

```diff
@@ -257,7 +264,8 @@
           "q8_0"
         ],
         "model_id": "TheBloke/vicuna-33B-GGML",
-        "model_file_name_template": "vicuna-33b.ggmlv3.{quantization}.bin"
+        "model_file_name_template": "vicuna-33b.ggmlv3.{quantization}.bin",
+        "model_revision": null
```

Comment:
Explicitly specifying null is unnecessary in this case.

Reply:
every model_revision: null is deleted in the latest version